### PR TITLE
Servlet and Fetch Tutorial

### PR DESCRIPTION
**Pull request by aamanhey**
Adds a getContent fetch function to get a message when a new button 'Get a message' in index.html us clicked. Changes the function called in the example to using arrow functions. Changes the message in the DataServlet in doGet()

### CODE REVIEW

**Review by saur-g — COMMENTED** [review]
Could you please deploy the app with these change, thanks.

**Comment by aamanhey** [comment]
I just deployed the app with the changes.

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (saur-g) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: Is this change needed?

**Review by saur-g — APPROVED** [review]
Goo work!  I didn't knew javascript had async functions, that's something cool to learn :)
